docs/submission/dictionary-overview.md

```diff
@@ -1,58 +1,55 @@
+### Field Descriptors
 
-### Field Name & Description
-Each field has a name and a description, the name being the same label that appears in the headers of each TSV file. 
+Each field has a data tier and an attribute classification, which reflects the importance of the field in terms of clinical data completion.
 
-### Data Tier & Attributes 
-Each field has a data tier and an attribute classification, which reflects the importance of the field in terms of clinical data completion. 
+![ID](/assets/submission/dictionary-id.svg) attribute indicates:
 
-#### ID Fields
-An ![ID](/assets/submission/dictionary-id.svg) field is a unique identifier that is used for cross file validation. Most ids have a ![Required](/assets/submission/dictionary-required.svg) attribute, which means they are required to be provided in an uploaded clinical file in order for a submission to be valid. 
+- An field is a unique identifier that is used for cross file validation.
+- This field is a primary or foreign key.
 
-#### Core Fields
-![Core](/assets/submission/dictionary-core.svg) clinical fields are required for each donor in order to be accepted as an ICGC ARGO case. The set of core clinical fields were defined by the [Tissue & Clinical Annotations Working Group](http://www.icgc-argo.org/page/84/tissue-clinical-annotation-working-group) which involved regular discussions with members of the working group and ARGO Programs. 
+![Conditional](/assets/submission/dictionary-conditional.svg) attribute indicates:
 
-These core clinical fields are commonly acquired in cohort-based studies and clinical trials and are required to address clinically relevant topics by cross entity analyses, and therefore constitute a critical element in the analysis of diverse ARGO Programs.
+- Field must meet certain conditions, depending on the value of another field.
+- Conditions described in the data dictionary scripts & notes.
 
-As seen in the dictionary, most core fields are ![Required](/assets/submission/dictionary-required.svg) for [clinical data completion](clinical-data-validation-rules), with the exception of some conditional fields that are dependent on the values of other fields. Upon upload, a validation error will occur, such as *"vital_status is a required field"*, if a core field is missing.
+![Required](/assets/submission/dictionary-required.svg) attribute indicates:
 
-If the field is ![Core](/assets/submission/dictionary-core.svg) + ![Conditional](/assets/submission/dictionary-conditional.svg), then it is required for clinical data completion, only if certain conditions are met. The conditions for the permissible value will be described in the notes column for the conditional field. 
+- Field must be filled in the submitted TSV file.
+- When pared with the `Conditional` attribute, the field is only required if conditional requirements are met.
```

Comment:
- When pared with the `Conditional` attribute, this field is only required if conditional requirements are met.